src/src/com/microsoft/aad/adal/AuthenticationActivity.java

```diff
@@ -790,7 +785,7 @@ protected TokenTaskResult doInBackground(String... urlItems) {
                 // Record account in the AccountManager service
                 try {
                     setAccount(result);
-                } catch (Exception exc) {
+                } catch (GeneralSecurityException | IOException exc) {
```

Comment:
I know java supports multi-catch, but I also want it to be super clear in the log that we can tell what is the exception thrown. I don't mind having a couple of more lines by just logging 'GeneralSecurityException is thrown'